proposals/2022-04-upgrade-v2-0-3-min-commission-rate/submit.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+panacead tx gov submit-proposal software-upgrade v2.0.3 \
+	--upgrade-time "2022-05-11T07:00:00Z" \
+	--node https://rpc.gopanacea.org:443 \
+	--chain-id panacea-3 \
+	--from panacea1ewugvs354xput6xydl5cd5tvkzcuymkejekwk3 \
+	--fees 1000000umed \
+	--gas auto \
+	-b block \
+	--deposit "10000000000umed" \
+	--title "Upgrade the chain to v2.0.3 for introducing a min-commission-rate parameter and setting it to 3%" \
+	--description "### Proposal
+	
+This proposal is for upgrading the Panacea mainnet 'panacea-3' from v2.0.2 to v2.0.3 for introducing a min-commission-rate parameter and setting it to 3%.
+
+A signaling proposal for this had been already approved: https://www.mintscan.io/medibloc/proposals/2.
```

Comment:
```suggestion
The signaling proposal for this change has already been approved: https://www.mintscan.io/medibloc/proposals/2.
```

Reply:
It seems that the proposal link needs to be changed.
https://www.mintscan.io/medibloc/proposals/7

Reply:
b824642f41a69adcdd0b719ad04e0a5ac7f7e51a thank you. gyuguen.